x-pack/test_serverless/functional/test_suites/security/ftr/navigation.ts

```diff
@@ -74,5 +74,13 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
         await testSubjects.existOrFail('cases-all-title');
       });
     });
+    it('navigates to maintenance windows', async () => {
+      await svlCommonPage.loginAsAdmin();
+      await svlSecNavigation.navigateToLandingPage();
```

Comment:
I think we do not need them as we do in o11y.

Reply:
This is needed due to permissions.